### PR TITLE
11.0.2

### PR DESCRIPTION
**Pull request by Xorban**
Pre patch fix

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (MSchiavi) [inline .vscode/settings.json:1]: Can we remove this from the PR

File: TrueStatValues
Comment (MSchiavi) [inline TrueStatValues:1]: This shouldn't be here as well

File: TrueStatValues.toc [hunk 2]
```diff
@@ -5,6 +5,10 @@
 ## Notes: Show Secondary-Stat Diminishing Returns in Tooltips.
 ## DefaultState: Enabled
 ## SavedVariables: TSV_DB 
+<<<<<<< HEAD
```
Comment (MSchiavi) [inline TrueStatValues.toc:8]: Is this for the icon?

File: TrueStatvalues.tga
Comment (MSchiavi) [inline TrueStatvalues.tga:1]: I am guessing this is the icon you were letting me know about?

**Review by MSchiavi — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Xorban) [reply .vscode/settings.json:1]: yes we can

Comment (Xorban) [reply .vscode/settings.json:1]: > Can we remove this from the PR

yes we can

**Review by MSchiavi — APPROVED** [review]

**Pull request merged**